src/sentry/api/bases/organization.py

```diff
@@ -312,17 +316,22 @@ def get_filter_params(
             "<10" if len_projects < 10 else "<100" if len_projects < 100 else ">100",
         )
 
-        environments = self.get_environments(request, organization)
         params = {
             "start": start,
             "end": end,
             "project_id": [p.id for p in projects],
             "organization_id": organization.id,
         }
+
+        environments = self.get_environments(request, organization)
         if environments:
             params["environment"] = [env.name for env in environments]
             params["environment_objects"] = environments
 
+        teams = self.get_teams(request, organization)
+        if teams:
+            params["team_id"] = [team.id for team in teams]
```

Comment:
Should we always set `params["team_id"]` so that we don't need to check the key exists?

Reply:
leaving this unset to match the convention with `"environment"`